packages/ceramic-core/src/pin/mock/mock-pinning-service.ts

```diff
@@ -0,0 +1,53 @@
+import { DocState } from "../../document"
+import PinningService from "../pinning-service"
+
+export default class MockPinningService implements PinningService {
```

Comment:
Is this mock pinning service really needed? Seems like it could be used for testing, but other than that it would only be bloating the codebase.

Reply:
In the tests we can probably just automock it using jest.